spec/cache/senders/impressions_sender_spec.rb

```diff
@@ -7,6 +7,7 @@
     let(:repository) { SplitIoClient::Cache::Repositories::ImpressionsRepository.new(adapter, config) }
     let(:sender) { described_class.new(repository, config, nil) }
     let(:formatted_impressions) { sender.send(:formatted_impressions, repository.clear) }
+    let(:ip) { Socket.ip_address_list.detect { |intf| intf.ipv4_private? }.ip_address }
```

Comment:
@sarrubia we return `unknow`: https://github.com/splitio/ruby-client/blob/fix/include_instance_id_into_namespace/lib/splitclient-rb/split_config.rb#L321